### ISSUE TITLE
Selector sub-properties with `configurable: false` should not show in form

### ISSUE THREAD

**Issue by mboldt** [cf-platform-eng/tile-generator#273]
With a form defined like:

```
forms:
- name: catalog_form
  label: Catalog
  description: Catalog configuration.
  properties:
  - name: catalog_presentation
    label: Customize catalog presentation
    description: Customize service presentation in the PCF marketplace
    default: "No"
    configurable: true
    type: selector
    option_templates:
    - name: default_service_presentation
      select_value: "No"
      property_blueprints:
      - name: display_name
        default: Bucket
        type: string
        label: Bucket
        configurable: false
    - name: custom_service_presentation
      select_value: "Yes"
      property_blueprints:
      - name: display_name
        default: Bucket
        type: string
        label: Bucket
        configurable: true
```

The tile configuration form shows the field for `display_name` when the "No" option is selected.

One would expect that, since that sub-property is not configurable (i.e., `configurable: false`), the field should not be shown in the form.

**Comment by mboldt** [comment]
This looks like the code that does the rendering: https://github.com/cf-platform-eng/tile-generator/blob/28731493b6b26157c8dc4fafc538eb5db075dadd/tile_generator/tile_metadata.py#L210-L221

**Comment by mboldt** [comment]
Fix will make it into tile-generator 12.0.7.
